### ISSUE TITLE
Practical 2 is too long

### ISSUE THREAD

**Issue by roddypr** [wurmlab/SBC361-programming-in-R#17]
Most students become stuck at question 14, few go on to the "working with DNA" section

**Comment by EmelineAurore** [comment]
I agree. Possible alternatives:

- change all questions after 14 for optional bonus/"hacker" questions (this will keep busy the most able, and the rest can do it at home)
- shorten the previous sections (remove Q4, Q9, Q10)

**Comment by yannickwurm** [comment]
Yes please do this, but please also add a pen and paper question
